RescheduleAppointments/force-app/main/default/lwc/mobileAppointmentBookingCalendar/mobileAppointmentBookingCalendar.js

```diff
@@ -183,20 +183,22 @@ export default class MobileAppointmentBookingRebookingCalendar extends Lightning
    */
 
   showWeekView(firstdayOfWeek) {
+    var firstArr = [];
+    var tempRow = [];
```

Comment:
Noticing a mix of using `var` here, vs. tacking away from it with `let` in another page below. Seems like it could be good to standardize, probably on the latter.

Reply:
@khawkins same as the comment above.